### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by timbeiko**
Signed-off-by: Tim Beiko <t.beiko23@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,14 +24,13 @@ and feel free to propose changes to this document in a pull request.
...
-you are agreeing to uphold this code. Please report unacceptable behavior. Please report unacceptable behavior to [private@pegasys.tech].
+you are agreeing to uphold this code. Please report unacceptable behavior. Please report unacceptable behavior to [private-quorum@consensys.net].
 
 ## I just have a quick question
 
 > **Note:** Please don't file an issue to ask a question.  You'll get faster results by using the resources below.
 
-* [permissioning-smart-contracts documentation]
-* [Gitter]
+* [RocketChat]
```
Comment (arash009) [inline CONTRIBUTING.md:33]: These arent part of the HLF, should comms be directed to the ConsenSys Discord instead?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -96,6 +95,6 @@ bigger changes should contain more detail.
 * Provide a summary on the first line with more details on additional lines as needed
 * Reference issues and pull requests liberally
 
-[private@pegasys.tech]: mailto:private@pegasys.tech
-[Gitter]: https://gitter.im/PegaSysEng/pantheon
+[private-quorum@consensys.net]: mailto:private-quorum@consensys.net
+[RocketChat]: https://chat.hyperledger.org/channel/besu
```
Comment (arash009) [inline CONTRIBUTING.md:99]: same here

**Comment by timbeiko** [comment]
Good catch, @arash009! I wasn't sure about RC. Changed the links to Discord :+1:

**Review by macfarla — APPROVED** [review]
LGTM

**Pull request merged**